docs/dataset/formatting-your-datasets.md

```diff
@@ -129,21 +186,28 @@ df_merged = df_metadata.merge(df_boxes, on="locator")
 dataframe_to_csv(df_merged, "processed.csv")
 ```
 
+The file `processed.csv` can be uploaded through the [:kolena-dataset-16: Datasets](https://app.kolena.io/redirect/datasets)
+page.
+
 ### Formatting results for Object Detection
 
 For Object Detection problems, model results need to have the following columns
 for the best experience. The values for each of the columns is a [`List[ScoredLabeledBoundingBox]`](../reference/annotation.md#kolena.annotation.ScoredLabeledBoundingBox)
 
 | Column Name            | Description                                         |
 |------------------------|-----------------------------------------------------|
-| matched_inference      | Inferences that were matched to a ground truth.     |
-| unmatched_inference    | Inferences that were not matched to a ground truth. |
-| unmatched_ground_truth | Inferences that were not matched to a ground truth. |
+| `matched_inference`      | Inferences that were matched to a ground truth.     |
+| `unmatched_inference`    | Inferences that were not matched to a ground truth. |
+| `unmatched_ground_truth` | Inferences that were not matched to a ground truth. |
 
 These columns are used to determine `True Postitives`, `False Positives`, and `False Negatives`.
+These results can be formated for upload with a similar process as above. This is done by adding the relavent list of
```

Comment:
`formated` -> `formatted`, `relavent` -> `relevant`. Can we run a spell checker over the changes?

Reply:
Ran a spell checker over the changes and updated. In a follow up going to look at adding a spell check to the pre-commit for docs.